dev-vcs/tortoisehg/tortoisehg-5.3.2.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit desktop distutils-r1
+
+if [[ ${PV} != *9999* ]]; then
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc64 ~sparc ~x86"
+	SRC_URI="https://www.bitbucket.org/${PN}/targz/downloads/${P}.tar.gz"
+	HG_DEPEND=" >=dev-vcs/mercurial-5.3.1
+		<dev-vcs/mercurial-5.4"
+else
+	inherit mercurial
+	EHG_REPO_URI="https://bitbucket.org/tortoisehg/thg"
+	EHG_REVISION="stable"
+	HG_DEPEND="dev-vcs/mercurial"
+fi
+
+DESCRIPTION="Set of graphical tools for Mercurial"
+HOMEPAGE="https://tortoisehg.bitbucket.io/"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="doc"
+
+RDEPEND="${HG_DEPEND}
+	$(python_gen_cond_dep '
+		dev-python/iniparse[${PYTHON_USEDEP}]
+		dev-python/pygments[${PYTHON_USEDEP}]
+		dev-python/PyQt5[network,svg,${PYTHON_USEDEP}]
+		>=dev-python/qscintilla-python-2.9.4:=[qt5(+),${PYTHON_USEDEP}]
+	')"
+DEPEND="${RDEPEND}
+	doc? (
```

Comment:
There's `distutils_enable_sphinx` function that does all of doc building and installing correctly.

Reply:
Thanks, I hadn't known about it before